examples/aws/terraform/dynamo.tf

```diff
@@ -25,6 +25,9 @@ resource "aws_dynamodb_table" "teleport" {
     type = "S"
   }
 
+  stream_enabled = "true"
+  stream_view_type = "NEW_AND_OLD_IMAGES"
```

Comment:
Does it need to be NEW and OLD or can it just be New? 

I set it as `NEW` in the AWS Guide 
https://gravitational.com/teleport/docs/aws_oss_guide/#dynamodb

Reply:
Good question, I actually don't know. @klizhentas?